Makefile.am

```diff
@@ -27,21 +27,20 @@ ACLOCAL_AMFLAGS	= -I m4
 ## libsemigroups headers
```

Comment:
Can you change the entries in `Makefile.am` so that there is only one file per line, and they are in alphabetical order after `libsemigroups-config.h` and `libsemigroups-debug.h`? It is too large a pain to keep them in order otherwise :)